pkg/config/toml/config.go

```diff
@@ -628,9 +628,12 @@ func (a *Automation) setFrom(f *Automation) {
 }
 
 type Workflow struct {
-	FromAddress      *types.EIP55Address `toml:",omitempty"`
-	ForwarderAddress *types.EIP55Address `toml:",omitempty"`
-	GasLimitDefault  *uint64
+	FromAddress       *types.EIP55Address `toml:",omitempty"`
+	ForwarderAddress  *types.EIP55Address `toml:",omitempty"`
+	GasLimitDefault   *uint64
+	TxAcceptanceState *commontypes.TransactionStatus
+	PollPeriod        *commonconfig.Duration
+	AcceptanceTimeout *commonconfig.Duration
```

Comment:
Let's sort these alphabetically so they are easier to look up in the generated docs